app/ConfigRepository.php

```diff
@@ -293,6 +289,11 @@ public function has($key): bool
             return false;
         }
 
+        // Check if we need to load the OS YAML file
+        if (preg_match('/^os\.(?<os>[^.]+)/', $key, $matches)) {
```

Comment:
Probably could be optimized a little with starts_with() and substr()

Reply:
Done